nssp_demo/postprocess_state_forecast.R

```diff
@@ -24,7 +25,7 @@ purrr::walk(script_packages, \(pkg) {
 
 
 # To be replaced with reading tidy data from forecasttools
```

Comment:
In the meantime, this should probably be moved to it's own script. In that script we can save each group as a separate `parquet` (as would be done with an equivalent python script) and then load only the necessary parquets here.

Reply:
Created an issue: https://github.com/CDCgov/pyrenew-hew/issues/107